docker-compose.yaml

```diff
@@ -68,7 +71,7 @@ services:
       POSTGRES_PASSWORD: *db_pass
       POSTGRES_DB: *db_name
     ports:
-      - 15432:5432
+      - 5432:5432
```

Comment:
We need a different port because some postgresql may already run on the dev machine.

Reply:
Addressed in https://github.com/swissgeol/ngm/pull/827/commits/64d2e491ea8b53a42bd1f8ab13bee9b45c89a357